andela/bad_ass-ah-frontend
# #166266937 Ft create and view comment

**What does this PR do?**

- This PR will allow a given user to post and view comments

**Description of Task to be completed?**

- A given User will be able to post a comment to an article

- Given User will be able to view all comments on a specific article

**How should this be manually tested?**

- Make sure that the server is running if not run the command `npm run dev`

- Provide credentials of a registered user and login to the system

- From `Hompage` click on a single article

- Comment to that article

**What are the relevant pivotal tracker stories?**

[#166266937](https://www.pivotaltracker.com/story/show/166266937)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/46047244/58978223-af429d00-87cb-11e9-97f1-63eeb683ace7.png)
![image](https://user-images.githubusercontent.com/46047244/58978233-b49fe780-87cb-11e9-8435-33d04642f6c6.png)
